### PR TITLE
[10.0][IMP] account_payment_return: Add return expenses functionality

### PR DESCRIPTION
**Pull request by luismontalba**
Improvements:
* Adds commissions expenses on each return line. When confirming return the expense is incorporated to the created move.
* A link to the move created from the return view form.

cc @Tecnativa

### CODE REVIEW

File: account_payment_return/models/payment_return.py [hunk 1]
```diff
@@ -157,6 +158,28 @@ def action_confirm(self):
                 (move_line | move_line2).reconcile()
                 return_line.move_line_ids.mapped('matched_debit_ids').write(
                     {'origin_returned_move_ids': [(6, 0, returned_moves.ids)]})
+            if return_line.expense_amount and return_line.expense_account:
```
Comment (pedrobaeza) [inline account_payment_return/models/payment_return.py]: Both expressions are the same.

File: account_payment_return/models/payment_return.py [hunk 2]
```diff
@@ -231,6 +254,13 @@ class PaymentReturnLine(models.Model):
         string='Amount',
         help="Returned amount. Can be different from the move amount",
         digits=dp.get_precision('Account'))
+    expense_account = fields.Many2one(
```
Comment (pedrobaeza) [inline account_payment_return/models/payment_return.py:257]: Put at journal level a default expense account and expense partner, and add an `_onchange_expense_amount` to bring this data to the fields.

File: account_payment_return/views/payment_return_view.xml [hunk 3]
```diff
@@ -42,6 +43,12 @@
                                                     ('reconciled', '=', True)]"
                                             />
                                     <field name="amount"/>
+                                    <field name="expense_account"
+                                           attrs="{'required':[('expense_amount','!=', 0)]}" />
+                                    <field name="expense_amount"
```
Comment (pedrobaeza) [inline account_payment_return/views/payment_return_view.xml]: Put it first and without `attrs`.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by luismontalba** [comment]
Changes proposed done in last commit.

File: account_payment_return/views/account_journal_view.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Luis M. Ontalba <luis.martínez@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3). -->
+
+<odoo>
+
+    <record id="view_account_journal_form" model="ir.ui.view">
+        <field name="name">account.journal.form</field>
+        <field name="model">account.journal</field>
+        <field name="type">form</field>
```
Comment (pedrobaeza) [inline account_payment_return/views/account_journal_view.xml]: Remove this deprecated key

File: account_payment_return/models/payment_return.py [hunk 5]
```diff
@@ -281,6 +281,13 @@ def _get_partner_from_move(self):
     def _onchange_move_line(self):
         self._compute_amount()
 
+    @api.onchange('expense_amount')
+    def _onchange_expense_amount(self):
+        journal = self.return_id.journal_id
```
Comment (pedrobaeza) [inline account_payment_return/models/payment_return.py]: Put this inside `if`

**Review by pedrobaeza — APPROVED** [review]
Little comments.

**Comment by luismontalba** [comment]
Changes done (commit squashed).

**Comment by pedrobaeza** [comment]
@cubells please review

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test.

**Pull request merged**